till_tech/www/lib/ionic/scss/_scaffolding.scss

```diff
@@ -0,0 +1,269 @@
+  margin: 0;
+  padding: 0;
+
+  color: $base-color;
+  word-wrap: break-word;
+  font-size: $font-size-base;
+  font-family: $font-family-base;
+  line-height: $line-height-computed;
+  text-rendering: optimizeLegibility;
+  -webkit-backface-visibility: hidden;
+  -webkit-user-drag: none;
+  -ms-content-zooming: none;
+}
+
+body.grade-b,
+body.grade-c {
+  // disable optimizeLegibility for low end devices
+  text-rendering: auto;
+}
+
+.content {
+  // used for content areas not using the content directive
+  position: relative;
+}
+
+.scroll-content {
+  position: absolute;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  overflow: hidden;
+
+  // Hide the top border if any
+  margin-top: -1px;
+
+  // Prevents any distortion of lines
+  padding-top: 1px;
+  margin-bottom: -1px;
+
+  width: auto;
+  height: auto;
+}
+
+.menu .scroll-content.scroll-content-false{
+  z-index: $z-index-scroll-content-false;
+}
+
+.scroll-view {
+  position: relative;
+  display: block;
+  overflow: hidden;
+
+  // Hide the top border if any
+  margin-top: -1px;
+}
+
+/**
+ * Scroll is the scroll view component available for complex and custom
+ * scroll view functionality.
+ */
+.scroll {
+  @include user-select(none);
+  @include touch-callout(none);
+  @include text-size-adjust(none);
+  @include transform-origin(left, top);
+}
+/**
+ * Set ms-viewport to prevent MS "page squish" and allow fluid scrolling
+ * https://msdn.microsoft.com/en-us/library/ie/hh869615(v=vs.85).aspx
+ */
+@-ms-viewport { width: device-width; }
+
+// Scroll bar styles
+.scroll-bar {
+  position: absolute;
+  z-index: $z-index-scroll-bar;
+}
+// hide the scroll-bar during animations
+.ng-animate .scroll-bar {
+  visibility: hidden;
+}
+.scroll-bar-h {
+  right: 2px;
+  bottom: 3px;
+  left: 2px;
+  height: 3px;
+
+  .scroll-bar-indicator {
+    height: 100%;
+  }
+}
+
+.scroll-bar-v {
+  top: 2px;
+  right: 3px;
+  bottom: 2px;
+  width: 3px;
+
+  .scroll-bar-indicator {
+    width: 100%;
+  }
+}
+.scroll-bar-indicator {
+  position: absolute;
+  border-radius: 4px;
+  background: rgba(0,0,0,0.3);
+  opacity: 1;
+  @include transition(opacity .3s linear);
+
+  &.scroll-bar-fade-out {
+    opacity: 0;
+  }
+}
+.platform-android .scroll-bar-indicator {
+  // android doesn't have rounded ends on scrollbar
+  border-radius: 0;
+}
+.grade-b .scroll-bar-indicator,
+.grade-c .scroll-bar-indicator {
+  // disable rgba background and border radius for low end devices
+  background: #aaa;
+
+  &.scroll-bar-fade-out {
+    @include transition(none);
+  }
+}
+
+ion-infinite-scroll {
+  height: 60px;
+  width: 100%;
+  display: block;
+
+  @include display-flex();
+  @include flex-direction(row);
+  @include justify-content(center);
+  @include align-items(center);
+
+  .icon {
+    color: #666666;
+    font-size: 30px;
+    color: $scroll-refresh-icon-color;
+  }
+  &:not(.active){
+    .spinner,
+    .icon:before{
+      display:none;
+    }
+  }
+}
+
+.overflow-scroll {
+  overflow-x: hidden;
```

Comment:
Properties should be ordered bottom, left, -webkit-overflow-scrolling, overflow-x, overflow-y, position, right, top